packages/trampoline/src/trampoline.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable @jessie.js/safe-await-separator */
+/**
+ * @import {TrampolineResult, SyncTrampolineResult} from './types.js'
+ */
+const { getPrototypeOf } = Object;
+const { bind } = Function.prototype;
+const uncurryThis = bind.bind(bind.call); // eslint-disable-line @endo/no-polymorphic-call
```

Comment:
Mind that the rule might not be enabled for this package (and probably should be enabled)

Reply:
the eslint rule that warns if eslint directives are unused should be enabled 😄